src/Examples/TensorboardExample/Tensorboard.cs

```diff
@@ -0,0 +1,92 @@
+            writer.add_text("AddText", "step_1", 1);
+            writer.add_text("AddText", "step_2", 2);
+            writer.add_text("AddText", "step_3", 3);
+            writer.add_text("AddText", "step_4", 4);
+            writer.add_text("AddText", "step_5", 5);
+        }
+
+        private static void AddImageUsePath(Modules.SummaryWriter writer)
+        {
+            string[] imagesPath = Directory.GetFiles("TensorboardExample/Images");
+            for (int i = 0; i < imagesPath.Length; i++) {
+                writer.add_img("AddImageUsePath", imagesPath[i], i);
+            }
+        }
+
+        private static void AddImageUseTensor(Modules.SummaryWriter writer, Device device)
+        {
+            SKBitmap[] images = Directory.GetFiles("TensorboardExample/Images").Select(item => SKBitmap.Decode(item)).ToArray();
+            using var d = NewDisposeScope();
+            for (int i = 0; i < images.Length; i++) {
+                Tensor tensor = SKBitmapToTensor(images[i], device);
+                writer.add_img("AddImageUseTensor", tensor, i, dataformats: "CHW");
+                images[i].Dispose();
+            }
+        }
+
+        private static void AddVideo(Modules.SummaryWriter writer, Device device)
+        {
+            SKBitmap[] images = Directory.GetFiles("TensorboardExample/Images").Select(item => SKBitmap.Decode(item)).ToArray();
+            using var d = NewDisposeScope();
+            Tensor tensor = stack(images.Select(item => SKBitmapToTensor(item, device)).ToArray());
+            tensor = stack(new Tensor[] { tensor, tensor, tensor, tensor, tensor, tensor, tensor, tensor, tensor, tensor });
+            foreach (var image in images)
+                image.Dispose();
+            writer.add_video("AddVideo", tensor, 1, 10);
+            writer.add_video("AddVideo", tensor, 2, 10);
+            writer.add_video("AddVideo", tensor, 3, 10);
+        }
+
+        private static Tensor SKBitmapToTensor(SKBitmap skBitmap, Device device)
+        {
+            int width = skBitmap.Width;
+            int height = skBitmap.Height;
+            byte[,,] hwcData = new byte[3, height, width];
+            for (int y = 0; y < height; y++) {
+                for (int x = 0; x < width; x++) {
+                    SKColor color = skBitmap.GetPixel(x, y);
+                    hwcData[0, y, x] = color.Red;
+                    hwcData[1, y, x] = color.Green;
+                    hwcData[2, y, x] = color.Blue;
+                }
+            }
+
+            //byte[,,] hwcData = new byte[4, height, width];
+            //for (int y = 0; y < height; y++) {
+            //    for (int x = 0; x < width; x++) {
+            //        SKColor color = skBitmap.GetPixel(x, y);
+            //        hwcData[0, y, x] = color.Red;
+            //        hwcData[1, y, x] = color.Green;
+            //        hwcData[2, y, x] = color.Blue;
+            //        hwcData[3, y, x] = color.Alpha;
+            //    }
+            //}
```

Comment:
If I change it to this, the program crashes.

A breakpoint instruction (__debugbreak() statement or similar call) has been executed in Examples.exe.

Reply:
@NiklasGustafsson 
I still can't find the reason for this problem, can you check it for me if you have some free time.

Reply:
When does it crash? After the conversion? All image operations on tensor assume that the number of channels is either 1 or 3. I don't think there's a single torch operator that knows about alpha channels.

Reply:
I suspect it has something to do with GC
If you put the breakpoint here, wait for the program to execute to this side. Press F11 and the program will crash immediately, the program does not enter the make_image(SKBitmap img, double rescale = 1) method

https://github.com/ChengYen-Tang/TorchSharp/blob/tensorboard/src/TorchSharp/Utils/tensorboard/Summary.cs#L84

Reply:
Confused -- the comment is in SKBitmapToTensor, but line 84 in Summary.cs is calling TensorToSKBitmap.

Reply:
Because their upper layer is this method

https://github.com/ChengYen-Tang/TorchSharp/blob/tensorboard/src/Examples/TensorboardExample/Tensorboard.cs#L46-L55